src/modules/sign-in/SignIn.jsx

```diff
@@ -63,10 +77,50 @@ export class SignIn extends Component {
         </Grid>
       )
     }
+  };
+
+  onChange = (e) => {
+   const name = e.target.name;
+   const value = e.target.value;
+   this.setState({[name]: value});
+  }
+
+  onBlur = (e) => {
+    const name = e.target.name;
```

Comment:
same here

Reply:
done
onBlur = (e) => {
    this.validateField(e.target.name);
    this.validateButtonState();
   }